<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>AskAnything</class>
  <widget class="QWidget" name="AskAnything">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>1366</width>
        <height>768</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>AskAnything</string>
    </property>
    <widget class="QLabel" name="label">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>1366</width>
          <height>768</height>
        </rect>
      </property>
      <property name="styleSheet">
        <string notr="true" />
      </property>
      <property name="text">
        <string />
      </property>
      <property name="pixmap">
        <pixmap resource="rec.qrc">lib/tuse.png</pixmap>
      </property>
      <property name="scaledContents">
        <bool>true</bool>
      </property>
    </widget>
    <widget class="QLabel" name="label_4">
      <property name="geometry">
        <rect>
          <x>500</x>
          <y>180</y>
          <width>650</width>
          <height>470</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="label_5">
      <property name="geometry">
        <rect>
          <x>40</x>
          <y>135</y>
          <width>200</width>
          <height>50</height>
        </rect>
      </property>
      <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#ffffff;&quot;&gt;23 OCT 2020&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
    </widget>
    <widget class="QPushButton" name="exitB">
      <property name="geometry">
        <rect>
          <x>1120</x>
          <y>660</y>
          <width>221</width>
          <height>71</height>
        </rect>
      </property>
      <property name="styleSheet">
        <string notr="true">background-image: url(lib/exit - Copy.png);
          border:none;</string>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>
    <widget class="QPushButton" name="startB">
      <property name="geometry">
        <rect>
          <x>1120</x>
          <y>50</y>
          <width>230</width>
          <height>103</height>
        </rect>
      </property>
      <property name="styleSheet">
        <string notr="true">background-image: url(lib/start.png);
          border:none;</string>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>
  </widget>
  <resources>
    <include location="rec.qrc" />
  </resources>
  <connections />
</ui>